contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -791,18 +784,23 @@ func (s *SchedulerServer) bootstrap(hks hyperkube.Interface, sc *schedcfg.Config
 	}
 
 	// create event recorder sending events to the "" namespace of the apiserver
+	eventsClientConfig := *clientConfig
+	eventsClient, err := clientset.NewForConfig(&eventsClientConfig)
+	if err != nil {
+		log.Fatalf("Invalid API configuration: %v", err)
+	}
 	broadcaster := record.NewBroadcaster()
 	recorder := broadcaster.NewRecorder(api.EventSource{Component: api.DefaultSchedulerName})
 	broadcaster.StartLogging(log.Infof)
-	broadcaster.StartRecordingToSink(client.Events(""))
+	broadcaster.StartRecordingToSink(&unversioned_core.EventSinkImpl{eventsClient.Events("")})
```

Comment:
it happens. in the future, please consider /cc'ing me for a quick look over changes in mesos code.